haskell/private/coverage_wrapper.sh.tpl

```diff
@@ -25,38 +27,40 @@ fi
 
 ERRORCOLOR='\033[1;31m'
 CLEARCOLOR='\033[0m'
-binary_path=$(rlocation {binary_path})
-hpc_path=$(rlocation {hpc_path})
-tix_file_path={tix_file_path}
-coverage_report_format={coverage_report_format}
-strict_coverage_analysis={strict_coverage_analysis}
-package_path={package_path}
+binary_path=$(rlocation "{binary_path}")
+hpc_path=$(rlocation "{hpc_path}")
+tix_file_path="{tix_file_path}"
+coverage_report_format="{coverage_report_format}"
+strict_coverage_analysis="{strict_coverage_analysis}"
+package_path="{package_path}"
 
 # either of the two expected coverage metrics should be set to -1 if they're meant to be unused
-expected_covered_expressions_percentage={expected_covered_expressions_percentage}
-expected_uncovered_expression_count={expected_uncovered_expression_count}
+expected_covered_expressions_percentage="{expected_covered_expressions_percentage}"
+expected_uncovered_expression_count="{expected_uncovered_expression_count}"
 
 # gather the hpc directories
 hpc_dir_args=""
-mix_file_paths={mix_file_paths}
+mix_file_paths="{mix_file_paths}"
```

Comment:
This looks fishy. This is substituted [with a shell array](https://github.com/tweag/rules_haskell/blob/4fe7b3a4e1b87717f5f081a7f574a96cd91f213c/haskell/private/haskell_impl.bzl#L300). But wrapping it in quotes will turn it from a shell array like `(a b c)` into a single string `"(a b c )"`. This should break the look in the next line. 

This may be causing the CI failure. The Darwin pipeline fails on the coverage step.

Reply:
I fixed it in ec51733dea8818e176202f007264b4f8f83dd9c0 as well as a similar one concerning `modules_to_exclude`. Thanks @aherrmann for the good catch!